source/future.tex

```diff
@@ -204,8 +204,6 @@
 as defaulted is deprecated if the class has
 a user-declared copy constructor or
 a user-declared destructor.
-In a future revision of \Cpp{}, these implicit definitions
-could become deleted\iref{dcl.fct.def.delete}.
```

Comment:
@zygoloid: Do you think this deletion is sufficiently editorial, and we're not deleting WG21 intent?

Reply:
In constructs like this, it seems adequately equivalent (and apparently ISO-legitimate) to say "It is possible that future versions of C++ will specify that these implicit definitions are deleted ([dcl.fct.def.delete]).", avoiding the question of changing the intent.

Reply:
Thanks, that sounds like a better solution!